src/dmangle.d

```diff
@@ -88,6 +94,9 @@ private immutable char[TMAX] mangleChar =
     //              X   // variadic T t...)
     //              Y   // variadic T t,...)
     //              Z   // not variadic, end of parameters
+    //              %   // Type backward reference
+    //              ?   // symbol backward reference
+    //              $   // identifier backward reference
```

Comment:
> BTW: currently 'Q' is unused, and mangling a TypeIdentifier ('I') doesn't make much sense for me, too.

You probably need a very good reason to pinch the last available character. ;-)

I think there may be some coverage tests for `I` in the gdb testsuite, but if you're right, then it should be a bug to see it in the backend.  But I have a suspicion that you may find it in some templates.

Maybe I shouldn't point at C++ as a way of example, but their way of compressing symbols seems to be good enough for them (and unquestionably portable).  We could use a system of `Q_`, `Q1_`, etc...